SETUP.md

```diff
@@ -2,7 +2,9 @@
 
 The easiest way to install Baritone is to install [Impact](https://impactclient.net/), which comes with Baritone.
 
-For 1.14.4, [click here](https://www.dropbox.com/s/rkml3hjokd3qv0m/1.14.4-Baritone.zip?dl=1).
+1.14.4 and lower is located on the [releases](https://github.com/cabaletta/baritone/releases/) page, if you don't want to use Impact. 
```

Comment:
Just add the links for 1.13, 1.14 and 1.15 versions then tell them to go to releases for forge 1.12???

Reply:
What? That is a link to them

Reply:
If I link directly to the release versions this file will have to be changed *every* time there is a new release, not just for major releases.

Reply:
Plus, isn't 1.15.2 ALSO on the releases page xD

Reply:
Now it is I guess lol